lab_3/main.py

```diff
@@ -59,49 +85,98 @@ def encode_corpus(storage: LetterStorage, corpus: tuple) -> tuple:
     :param corpus: a tuple of sentences
     :return: a tuple of the encoded sentences
     """
-    pass
+    if not isinstance(storage, LetterStorage) or not isinstance(corpus, tuple):
+        return ()
+
+    encoded_corpus = []
+    for sentence in corpus:
+        list_sentence = []
+        if isinstance(sentence[0], tuple):
```

Comment:
why is this check needed?

Reply:
I couldn't pass all the tests without it, but I pulled updates yesterday, so now all is working and I will remove it